From 4a1b8659235dcac5382a627a4add4d636a65a886 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Wed, 28 Dec 2022 16:31:05 +0800
Subject: [PATCH] wscript: skip checking PYTHONHASHSEED

Skip checking PYTHONHASHSEED as it is set to 0 in oe-core by default.

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 buildtools/wafsamba/wscript | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/buildtools/wafsamba/wscript b/buildtools/wafsamba/wscript
index 8729b08..a4d6f3e 100644
--- a/buildtools/wafsamba/wscript
+++ b/buildtools/wafsamba/wscript
@@ -8,10 +8,6 @@ import wafsamba
 from samba_utils import symlink
 from optparse import SUPPRESS_HELP
 
-phs = os.environ.get("PYTHONHASHSEED", None)
-if phs != "1":
-    raise Errors.WafError('''PYTHONHASHSEED=1 missing! Don't use waf directly, use ./configure and make!''')
-
 # this forces configure to be re-run if any of the configure
 # sections of the build scripts change. We have to check
 # for this in sys.argv as options have not yet been parsed when
-- 
2.25.1

